<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewClothRollDialog</class>
 <widget class="QDialog" name="NewClothRollDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>409</width>
    <height>337</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>新建布卷对话框</string>
  </property>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>10</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>新建布卷</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>381</width>
     <height>201</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>300</width>
     <height>0</height>
    </size>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>布卷编号：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="clothRollNoLE">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>10</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>80</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>机器编号：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="machineNoLE">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>70</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>140</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>操作工人：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="operaNameLE">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>130</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>270</y>
     <width>150</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>150</width>
     <height>50</height>
    </size>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>270</y>
     <width>150</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>150</width>
     <height>50</height>
    </size>
   </property>
   <property name="text">
    <string>确认</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked(bool)</signal>
   <receiver>NewClothRollDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>273</x>
     <y>186</y>
    </hint>
    <hint type="destinationlabel">
     <x>238</x>
     <y>115</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
